forecast_tools.R

```diff
@@ -86,8 +86,8 @@ plot_data = function(data) {
     text = getURL(
       "https://raw.githubusercontent.com/weecology/PortalData/master/Rodents/moon_dates.csv"))
   target_moon=unique(data$NewMoonNumber)
-  period_code = dplyr::filter(newmoons_table, newmoons_table$NewMoonNumber == target_moon) %>%
-    dplyr::select(Period)
+  period_code = as.integer(dplyr::filter(newmoons_table, newmoons_table$NewMoonNumber == target_moon) %>%
+    dplyr::select(Period))
```

Comment:
Yeah, sorry, that's me being sloppy and leaving out the remote. So assuming you only have one remote it would be:

```
git push origin repair_plot_data_func
```